CMakeLists.txt

```diff
@@ -4,8 +4,8 @@ option(USE_GPU "Enable GPU-accelerated training" OFF)
 option(USE_SWIG "Enable SWIG to generate Java API" OFF)
 option(USE_HDFS "Enable HDFS support (EXPERIMENTAL)" OFF)
 option(USE_TIMETAG "Set to ON to output time costs" OFF)
-option(USE_CUDA "Enable CUDA-accelerated training (EXPERIMENTAL)" OFF)
-option(USE_CUDA_EXP "Enable CUDA-accelerated training with more acceleration (EXPERIMENTAL)" OFF)
+option(USE_CUDA "Enable CUDA-accelerated training " OFF)
+option(USE_CUDA_EXP "(DEPRECATED) Alias for 'USE_CUDA'. Use 'USE_CUDA' instead." OFF)
```

Comment:
My initial proposal here was to keep it on `master` for a short time (maybe a few weeks) so that people cloning the repo without pinning to a specific commit would at least get an informative warning. But revisiting it I think you're right...we should just remove `CUDA_EXP` completely everywhere. I'll push another commit here doing that.

Reply:
I just pushed https://github.com/microsoft/LightGBM/pull/5677/commits/ac7ab774d5152478aa660028c2ee0d866a008daa completely removing remaining uses of `CUDA_EXP`.